registry.json

```diff
@@ -151,6 +151,20 @@
     "operatorName": "CONRAD",
     "github": "conraddit"
   },
+  {
+    "ticker": "BEGIN",
+    "poolId": "b59bf139701b8cdae5c997225f438db7603ef1d9fd9a1f62dc5821eb",
+    "social": {
+      "website": "https://begin.is"
+      "twitter": "BeginWallet",
+      "facebook": "beginwallet",
+      "instagram": "beginwallet",
+      "youtube": "beginwallet"
+    },
+    "telegram": "francisluz",
+    "operatorName": "B58 Finance",
```

Comment:
Please change this to be the operators real name or leave blank

Reply:
Thanks @Bez625, change done.